corehq/apps/enterprise/enterprise.py

```diff
@@ -561,3 +564,22 @@ def rows_for_domain(self, domain_obj):
         num_errors = sum([result['direction_count'] for result in results if result['error']])
 
         return [(domain_obj.name, num_sent, num_received, num_errors), ]
+
+
+class Enterprise2FAReport(EnterpriseReport):
+    title = gettext_lazy('Two Factor Authentication')
+    total_description = gettext_lazy('# of Domains not having 2FA enforced')
```

Comment:
User-facing text typically uses "project spaces" instead of "domains".

Reply:
Thank you! 8b0ce9411e65f8c5f210035238cb0e63f381b582